examples/fabric-bridge-app/linux/include/RpcClient.h

```diff
@@ -32,13 +32,24 @@ constexpr uint16_t kFabricAdminServerPort = 33001;
  */
 CHIP_ERROR InitRpcClient(uint16_t rpcServerPort);
 
+struct CommissioningWindowParams
```

Comment:
please make this a builder pattern. It should read code like: 

```cpp
OCW(
CommisioningWindowParams()
   .SetNodeId(1234)
   .SetDiscriminator(123)
   .SetIterations(100)
   )
```

- [ ] make sure all members have initializers (if you do not know or they are required, set to 0 just in case)
- [ ] make this fluent builders to make it readable and not require pre-declaration
- [ ] It seems that @bzbarsky-apple mentioned some things are not supposed to work together. Add tests for that in the setters (probably VerifyOrDie is fine to avoid mis-use ... but at a minimum class coments describing use should exist here)

Reply:
Added fluent builder pattern, initialized some of the member variables, and added some commentary in https://github.com/project-chip/connectedhomeip/pull/34209/commits/16bf72cd708ca52afceff75c669f65e99068489b, LMK if we need to go further in this route.